Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/ui/person/ContactDiaryAddPersonViewModel.kt

```diff
@@ -90,11 +105,11 @@ class ContactDiaryPersonBottomSheetDialogViewModel @AssistedInject constructor(
 
     companion object {
         private const val MAX_PERSON_NAME_LENGTH = 250
-        private val TAG = ContactDiaryPersonBottomSheetDialogViewModel::class.java.simpleName
+        private val TAG = ContactDiaryAddPersonViewModel::class.java.simpleName
     }
 
     @AssistedFactory
-    interface Factory : CWAViewModelFactory<ContactDiaryPersonBottomSheetDialogViewModel> {
-        fun create(addedAt: String?): ContactDiaryPersonBottomSheetDialogViewModel
+    interface Factory : CWAViewModelFactory<ContactDiaryAddPersonViewModel> {
+        fun create(addedAt: String?): ContactDiaryAddPersonViewModel
```

Comment:
You didn't add this, but do you know where this is used?

Reply:
I can see it's an extra parameter for the fragment. You can see it in the navgraph.